packages/flutter/lib/src/services/platform_views.dart

```diff
@@ -292,6 +307,41 @@ class PlatformViewsService {
     }
     return AppKitViewController._(id, layoutDirection);
   }
+
+  /// Factory method to create a Win32ViewController.
+  static Future<Win32ViewController> initWindowsView({
+    required int id,
+    required String viewType,
+    VoidCallback? onFocus,
+    LoseFocusCallback? onLoseFocus,
+  }) async {
+    final Map<String, dynamic> args = <String, dynamic>{
+      'id': id,
+      'viewType': viewType,
+    };
+    await SystemChannels.platform_views.invokeMethod<void>('create', args);
+    if (onFocus != null) {
+      _instance._focusCallbacks[id] = onFocus;
+    }
+    if (onLoseFocus != null) {
+      _instance._loseFocusCallbacks[id] = onLoseFocus;
+    }
+    return Win32ViewController._(id);
+  }
+
+  /// Request that keyboard focus move to the provided platform view.
```

Comment:
"Request" meaning it can fail? How would success or failure be communicated back?

Reply:
This "shouldn't" fail as long as it is invoked with a valid platform view ID. If it is not, the method handler on the engine side will invoke `Error`, and it will invoke `Success` if it succeeds.